docs/wiki/end-user-guide/preparing-your-modpack.md

```diff
@@ -0,0 +1,94 @@
+
+To prepare your modpack for Cleanroom Loader, review the lists below and make replacements where necessary.
+
+## Deprecated / Incompatible Mods
+
+| Mod                             | Reason                                                      | Action                                                                                                                                                                                                     |
+|---------------------------------|-------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **AdvancedShader**              | Binary patching, incompatible.                              | :no_entry_sign: Remove                                                                                                                                                                                     |
+| **Essential.gg**                | Patched but causes many issues.                             | :white_check_mark: Use **[Lan UPNP](https://www.curseforge.com/minecraft/mc-mods/lan-upnp)** (Non-Proxy option)or **[E4MC Retro](https://www.curseforge.com/minecraft/mc-mods/e4mc-retro)** (Proxy Option) |
+| **FoamFix**                     | Outdated and prone to cause issues in Cleanroom Environment | :white_check_mark: Use **[VintageFix](https://www.curseforge.com/minecraft/mc-mods/vintagefix)**                                                                                                           |
+| **Forgelin**                    | Outdated and blacklisted                                    | :white_check_mark: Use **[Forgelin-Continuous](https://www.curseforge.com/minecraft/mc-mods/forgelin-continuous)**                                                                                         |
+| **Gnetum**                      | Prone to cause issues                                       | :white_check_mark: Use **[StellarCore](https://www.curseforge.com/minecraft/mc-mods/stellarcore)**                                                                                                         |
+| **JustEnoughIDs (JEID) / NEID** | Outdated                                                    | :white_check_mark: Use **[RoughlyEnoughIDs (REID)](https://www.curseforge.com/minecraft/mc-mods/reid/files/6761278)**                                                                                      |
+| **JustEnoughItems (JEI)**       | Outdated                                                    | :white_check_mark: Use **[HadEnoughItems (HEI)](https://www.curseforge.com/minecraft/mc-mods/had-enough-items)**                                                                                           |
```

Comment:
given that both of these are marked as just "Outdated", i believe they belong down in the section below with Superseded mods. perhaps JEID/NEID is incompatible? should have its reason changed if so.